lib/order.rb

```diff
@@ -0,0 +1,37 @@
+# require 'take_away_menu'
+#
+class Order
+
+  attr_reader :add_item, :list_of_items, :menu, :item
+
+  def initialize
+    @list_of_items = []
+    @menu = TakeAwayMenu.new
+  end
+
+  def add_item(item)
+    selected_items = @menu.items.assoc(item)
+    @list_of_items << Hash[*selected_items]
```

Comment:
As I understand it, this code gets the item name and price for the item to add and puts them as a key and value in a hash.  It then adds that hash to the `@list_of_items` array.  There's a simpler way to do this using operations like this: `hash[item]` and `hash[key] = value` and `array << hash`.  Happy to talk about this if this comment isn't enough guidance!